### PR TITLE
[14.0][MIG] purchase_manual_delivery

### PR DESCRIPTION
**Pull request by ChrisOForgeFlow**
Standard Migration
@ForgeFlow

- [x] Allow behavior configuration on company
- [x] Add test related with this configuration

### CODE REVIEW

File: purchase_manual_delivery/models/res_company.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import fields, models
+
+
+class ResCompany(models.Model):
+
+    _inherit = "res.company"
+
+    purchase_manual_delivery = fields.Boolean(
+        string="Purchase manual delivery?", required=False
```
Comment (CasVissers-360ERP) [inline purchase_manual_delivery/models/res_company.py:11]: @ChrisOForgeFlow Should we not set default=True here? To make the module work directly?

**Comment by CasVissers-360ERP** [comment]
@ChrisOForgeFlow 
Functional test: 
https://drive.google.com/file/d/1ff3p_VNXmq-u6PC6rv7ogHwEXGO1yM6I/view

Seems something is not working in the wizard.

**Comment by JordiBForgeFlow** [comment]
@ChrisOForgeFlow can you try to reproduce the issue reported by Cas Vissers?

**Comment by ChrisOForgeFlow** [comment]
> @ChrisOForgeFlow Functional test: https://drive.google.com/file/d/1ff3p_VNXmq-u6PC6rv7ogHwEXGO1yM6I/view
> 
> Seems something is not working in the wizard.

Thanks for review, is resolved

**Review by CasVissers-360ERP — APPROVED** [review]
Functional review.

**Review by HviorForgeFlow — APPROVED** [review]
Code + Functional Review LGTM :+1:

**Pull request merged**